tools/fastqpreprocessing/src/input_options.cpp

```diff
@@ -102,6 +102,7 @@ InputOptionsFastqProcess readOptionsFastqProcess(int argc, char** argv)
     /* These options don’t set a flag.
        We distinguish them by their indices. */
     {"bam-size",            required_argument, 0, 'B'},
+    {"num-output-files",    required_argument, 0, 'OF'},
```

Comment:
OF is a little bit confusing since it could be 0F (zero vs O). I suggest usning nF

Reply:
Sounds good. I just changed it nF. Thank you!